lib/taurus/core/tango/tangodatabase.py

```diff
@@ -679,8 +679,12 @@ def __init__(self, host=None, port=None, parent=None):
 
         # Set host to fqdn
         host = socket.getfqdn(host)
+        try:
+            self.dbObj = Database(host, port)
```

Comment:
If a exception occurs,  `self.dbObj` is not defined. I would set it to `None` instead

Reply:
Agree with it